verticaltabs.js

```diff
@@ -457,6 +471,25 @@ VerticalTabs.prototype = {
 
     let enterTimeout = -1;
 
+    let exit = (event) => {
+      this.mouseExited();
+      if (enterTimeout > 0) {
+        window.clearTimeout(enterTimeout);
+        enterTimeout = -1;
+      }
+      if (mainWindow.getAttribute('tabspinned') !== 'true') {
+        if (!leftbox.contextMenuOpen){
+          leftbox.removeAttribute('expanded');
+        }
+
+        this.clearFind();
+        let tabsPopup = document.getElementById('alltabs-popup');
+        if (tabsPopup.state === 'open') {
+          tabsPopup.hidePopup();
+        }
+      }
+    };
+
     let enter = (event) => {
       this.mouseEntered();
       if (event.type === 'mouseenter' && leftbox.getAttribute('expanded') !== 'true') {
```

Comment:
If we're not calling this on `mousemove`, then we probably don't need to check for `event.type === 'mouseenter'` here…